lib/extty.ex

```diff
@@ -97,7 +97,8 @@ defmodule ExTTY do
   defp start_shell(state) do
     %{
       state
-      | group: :group.start(self(), shell_spawner(state), [{:echo, true}]),
+      | group:
+          :group.start(self(), shell_spawner(state), [{:echo, true}, {:expand_below, false}]),
```

Comment:
```suggestion
          :group.start(self(), shell_spawner(state), echo: true, expand_below: false),
```

I wonder if you pass keyword lists the Elixir way if the formatter will let you keep it on one line. :)